src/General/Alert/Alert.tsx

```diff
@@ -137,7 +137,7 @@ export interface Props {
   type: AlertType;
   /** Set message for Alert. */
   message: string;
-  /** A function to close Alert. */
+  /** A function to close Alert. onClose will be triggered if the close icon is clicked or the Alert has ran its autoClose duration */
```

Comment:
Also pressing the Enter or ESC keys, can refer to
https://github.com/glints-dev/glints-aries/blob/v4/src/General/Alert/Alert.tsx#L65